radio.k0swe.Kel_Agent.yml

```
@@ -0,0 +1,45 @@
+app-id: radio.k0swe.Kel_Agent
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+build-options:
+  append-path: /usr/lib/sdk/golang/bin
+  env:
+    - 'LDFLAGS='
+command: kel-agent
```

```suggestion
command: kel-agent
rename-desktop-file: radio.k0swe.kel_agent.Kel_Agent.desktop
rename-appdata-file: radio.k0swe.kel_agent.Kel_Agent.metainfo.xml
rename-icon: radio.k0swe.kel_agent.Kel_Agent
```

This is needed to match.